tests/pep8/config

```diff
@@ -0,0 +1,9 @@
+# Configuration file for pep8 checker (pycodestyle or pep8)
+
+[pycodestyle]
+max-line-length = 99
+# Ignored by default:
+#ignore = E121,E123,E126,E133,E226,E241,E242,E704,W503,W504,W505
```

Comment:
What is the reason to keep this here?

Reply:
So we know what pycodestyle ignores by default. But I can remove it, it can be found in "man pycodestyle"

Reply:
I would remove it.

Reply:
Done